sendcloud/sendcloud.php

```diff
@@ -56,8 +56,8 @@ public function __construct()
         $this->boostrap = true;
         $this->name = 'sendcloud';
         $this->tab = 'shipping_logistics';
-        $this->version = '1.2.0';
-        $this->author = 'SendCloud Global B.V.';
+        $this->version = '1.3.0';
```

Comment:
Given there is a `BREAKING CHANGE:`, should this version be bumped to `2.0.0` instead?